Freshli.Web/Controllers/AnalysisRequestsController.cs

```diff
@@ -32,7 +36,158 @@ public class AnalysisRequestsController : Controller {
     [HttpGet("{id}", Name = "ShowAnalysisRequest")]
     public IActionResult Show(Guid id) {
       var analysisRequest = _db.AnalysisRequests.Find(id);
-      return View(analysisRequest);
+
+      if (analysisRequest.Results != null)
+      {
+        var orderedResults = analysisRequest.Results.
+          OrderBy(r => r.Date).ToArray();
+        var dates = orderedResults.Select(r => r.Date);
+
+        var projectTotalOverTime = ProjectTotalOverTime(
+          analysisRequest,
+          dates,
+          orderedResults
+        );
+        var projectAverageOverTime = ProjectAverageOverTime(
+          analysisRequest,
+          dates,
+          orderedResults
+        );
+        var projectMaxOverTime = ProjectMaxOverTime(
+          analysisRequest,
+          dates,
+          orderedResults
+        );
+
+        var names = orderedResults.Select(
+          r => r.LibYearResult.PackageResults.
+            Select(p => p.Name).ToArray()).Aggregate(
+          new HashSet<string>(),
+          (hash, values) => {
+            foreach(var value in values) {
+              hash.Add(value);
+            }
+            return hash;
+          }
+        ).OrderBy(name => name);
+
+        var areaSeries = new List<Scatter>();
+
+        foreach (var name  in names) {
+          var projectData = orderedResults.
+            Where(r => r.LibYearResult.PackageResults.
+              Any(p => p.Name == name));
+          var projectDates = projectData.Select(r => r.Date);
+          var projectLibYearValues = projectData.
+            Select(r => r.LibYearResult.PackageResults.
+              First(p => p.Name == name).Value);
+
+          areaSeries.Add(new Scatter {
+            name = name,
+            x = projectDates,
+            y = projectLibYearValues
+          });
+        }
+
+        var chart = Chart.Plot(areaSeries.ToArray());
+        chart.WithLayout(new Layout.Layout {
+          title = "LibYear over time per dependency",
```

Comment:
Use title case to match other graph titles.

Reply:
Fixed in e0988a1.